kani-compiler/src/kani_middle/transform/check_uninit/mod.rs

```diff
@@ -426,3 +464,20 @@ pub fn resolve_mem_init_fn(fn_def: FnDef, layout_size: usize, associated_type: T
     )
     .unwrap()
 }
+
+/// Checks if the instance is a harness -- an entry point of Kani analysis.
+fn is_harness(instance: Instance, tcx: TyCtxt) -> bool {
```

Comment:
This function could be useful elsewhere, so it might make sense to move it up (not necessarily in this PR).

Reply:
I took a note, will do